applications/common/NTPClient.c

```diff
@@ -23,8 +23,8 @@
 #include <time.h>
 
 #include "common.h"
-#include "ntpclient.h"
-#include "..\include\zephyr\sys\byteorder.h"
```

Comment:
The correct path should be: 
#include "zephyr\sys\byteorder.h"

and in the include folder searching I have this configured;
${workspaceFolder}/../env/ncs/zephyr/**